tests/frontend/test_kibana_app/cypress/integration/step-definitions/settings/logs/backend-reload-logs.then.js

```diff
@@ -0,0 +1,6 @@
+Then('The backend response indicates that the logs are reloaded', () => {
+  cy.get('@apiCheck').should('not.be.null');
+  cy.wait('@apiCheck').then( (interceptedObject) => {
+    expect(interceptedObject.response.statusCode).to.equal(200);
```

Comment:
Maybe we could add the status code in a constant or use the following npm module: https://www.npmjs.com/package/http-status-codes

Reply:
I think its a great idea, but maybe we can implement them in a future pull request this one is packing a lot of different 
functionality